jobseekers.html

```diff
@@ -1,152 +1,153 @@
-			</ul>
-
-
-			<ul id="lowerNav">
-				<li><a href="#">Career Assessment</a></li>
-				<li><a href="#">Q&A Platform</a></li>
-				<li><a href="#">Job Hunting Solutions</a></li>
-				<li><a href="#">Job Board</a></li>
-				<li><a href="#">Resources Center</a></li>
-				<li><a href="#">LogIn/Rigster</a></li>
-			</ul>
-		</nav>
-	</header>
-
-	<section id="slideshow">
-
-			<a class="w3-btn-floating" style="position:absolute;top:45%;left:13%" onclick="plusDivs(-1)"><img class="arrows" src="images/arrows_white_left.png" /></a>
-               <a class="w3-btn-floating" style="position:absolute;top:45%;right:13%" onclick="plusDivs(1)"><img class="arrows" 
-               	src="images/arrows_white_right.png" /></a>
-		<div id="slides">
-		    <img class="mySlides" src="images/rollPic1.jpg" alt="Slide photo 1" />
-		    <img class="mySlides" src="images/rollPic2.jpg" alt="Slide photo 2" />
-		    <img class="mySlides" src="images/rollPic3.jpg" alt="Slide photo 3" />
-		    <img class="mySlides" src="images/rollPic4.jpg" alt="Slide photo 4" />
-      	       
-		</div>
-		   		   		
-	</section>	
-
-	<section id="middle">
+	<!DOCTYPE html>
+	<html>
+	<head>
+		<!-- created by Sherry, Vicent, Dennis -- updated>
+		<!-- code review by Zizuo -->
+		<title>mentorplans | Yanset Landing Page 2C</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial scale = 1" />
+		<link href="normalize.css" rel="stylesheet" type="text/css" />
+		<link href="style.css" rel="stylesheet" type="text/css" />
 		
-		<div id="onestop">
-			<img class="slash" src="images/slash.png" alt="a forward slash" />
-			<h2>One Stop Solution</h2>
-		</div>
-
-		<div id="svg">
-
-			<div id="path">
-			<figure>
-			<object data="images/pathFinder.svg" type="image/svg+xml">PathFinder</object>
-			<figcaption>Path Finder</figcaption>
-			</figure>
-			<p>Discover your best-fit job by testing your interest, abilities, values and personality</p>
+	</head>
+	<body>
+
+		<header>
+			<a href="http://www.yanset.com/home"><img src="images/header-17.jpg" alt="header background image"></a>
+
+			<nav id="navigator">
+				<ul id="upperNav">
+					<li><a href="jobseekers.html">For Job Seekers</a></li>
+					<li><a href="http://www.yansetmentorplans.com/#!yanset-landing-page-2b/qptmz" target="_blank">For Career Centers</a></li>
+				</ul>
+
+
+				<ul id="lowerNav">
+					<li><a href="#">Career Assessment</a></li>
+					<li><a href="#">Q&A Platform</a></li>
+					<li><a href="#">Job Hunting Solutions</a></li>
+					<li><a href="#">Job Board</a></li>
+					<li><a href="#">Resources Center</a></li>
+					<li><a href="#">LogIn/Rigster</a></li>
+				</ul>
+			</nav>
+		</header>
+
+		<section id="slideshow">
+
+				<a class="w3-btn-floating" style="position:absolute;top:45%;left:13%" onclick="plusDivs(-1)"><img class="arrows" src="images/arrows_white_left.png" /></a>
```

Comment:
it is better to bind event inside JQuery or JS instead of in element's attribute